geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/Transformers.scala

```diff
@@ -334,6 +334,7 @@ class DateFunctionFactory extends TransformerFunctionFactory {
   val basicDateTimeNoMillis       = StandardDateParser("basicDateTimeNoMillis", ISODateTimeFormat.basicDateTimeNoMillis().withZoneUTC())
   val dateHourMinuteSecondMillis  = StandardDateParser("dateHourMinuteSecondMillis", ISODateTimeFormat.dateHourMinuteSecondMillis().withZoneUTC())
   val millisToDate                = TransformerFn("millisToDate") { args => new Date(args(0).asInstanceOf[Long]) }
+  val secsToDate                  = TransformerFn("secsToDate") { args => new Date(args(0).asInstanceOf[Int].toLong * 1000L) }
```

Comment:
Probably in this file: https://github.com/nelias-ccri/geomesa/blob/fcr_transform_fn/geomesa-convert/geomesa-convert-common/src/test/scala/org/locationtech/geomesa/convert/common/TransformersTest.scala#L113